lib/Platform/Intl/PlatformIntlApple.mm

```diff
@@ -68,12 +208,69 @@
   return canonicalizeLocaleList(runtime, locales);
 }
 
+// Implementer note: This method corresponds roughly to
+// https://tc39.es/ecma402/#sup-string.prototype.tolocalelowercase
 vm::CallResult<std::u16string> toLocaleLowerCase(
     vm::Runtime *runtime,
     const std::vector<std::u16string> &locales,
     const std::u16string &str) {
-  return std::u16string(u"lowered");
+  NSString *nsStr = u16StringToNSString(str);
+
+  // 3. Let requestedLocales be ? CanonicalizeLocaleList(locales).
+  vm::CallResult<std::vector<std::u16string>> requestedLocales =
+      getCanonicalLocales(runtime, locales);
```

Comment:
```suggestion
      canonicalizeLocaleList(runtime, locales);
```

Reply:
Once I moved a portion of toLocaleLowerCase up to a new function, localeListToLocaleString, I wasn’t able to call canonicalizeLocaleList, but I could still call getCanonicalLocales and I’m not sure why. Also, I wondered why it is better to call one over the other, as they do the same thing.

Reply:
In C++, the declaration order of functions matters. So you can't call a function that is declared after the current function. In this case, you would need to move `localeListToLocaleString` below `canonicalizeLocaleList `. You can still call `getCanonicalLocales` though, because it is forward declared in `PlatformIntl.h`.

As for why one is preferred, it is just to match the language in the spec.